ludwig/schema/metadata/configs/combiners.yaml

```diff
@@ -236,6 +248,18 @@ ComparatorCombiner:
             most tasks.
         ui_display_name: Layer Weights Initializer
 ConcatCombiner:
+    type:
+        short_description:
+            Concatenates the outputs of all the encoders.
```

Comment:
Concatenates the outputs of all the encoders and passes the concatenated representation through a stack of fully connected layers.